processor/parse/magic-block-parser.js

```diff
@@ -201,10 +201,14 @@ function tokenize(eat, value) {
     }
     case 'embed': {
       const { title, url, html } = json;
-      json.provider = new URL(url).hostname
-        .split(/(?:www)?\./)
-        .filter(i => i)
-        .join('.');
+      try {
+        json.provider = new URL(url).hostname
```

Comment:
🪦 RIP [`URL.parse`][parse]; it's deprecated now, but the constructor is so much more strict…

[parse]: https://nodejs.org/api/url.html#urlparseurlstring-parsequerystring-slashesdenotehost